src/imitation/scripts/train_imitation.py

```diff
@@ -140,6 +141,41 @@ def dagger(
     return stats
 
 
+@train_imitation_ex.command
+def sqil(
+    sqil: Mapping[str, Any],
+    _run,
+    _rnd: np.random.Generator,
+) -> Mapping[str, Mapping[str, float]]:
+    custom_logger, log_dir = logging_ingredient.setup_logging()
+    expert_trajs = demonstrations.get_expert_trajectories()
+
+    with environment.make_venv() as venv:
+        sqil_trainer = SQIL(
+            venv=venv,
+            demonstrations=expert_trajs,
+            policy=sqil["policy_model"],
+            custom_logger=custom_logger,
+            rl_algo_class=sqil["rl_algo_class"],
```

Comment:
It'd be good to use the `rl` ingredient. Although this may be a bit tricky since that ingredient is designed to *make* the RL algorithm rather than pass class and keyword arguments in (which we need given how SQIL is currently designed, in order to ensure a custom replay buffer gets specified).

Reply:
I've added the RL ingredient without using any of the functions provided which make the RL algorithm, instead only using the configs. Let me know if that works.

Reply:
Yes, this seems like a good solution!